tests/out/shapes.txt

```diff
@@ -24,1751 +24,1751 @@ shape_id,shape_pt_lat,shape_pt_lon,shape_pt_sequence
-3153803,53.0554537,8.8442286,1688
-3153803,53.0543613,8.8409939,1689
-3153803,53.0525009,8.8400534,1690
-3153803,53.0374124,8.821674,1691
-3153803,53.032495,8.8398022,1692
-3170153,53.0394778,8.8155034,1693
-3170153,53.0286287,8.8066292,1694
-3185196,53.0602695,8.7641142,1695
-3185196,53.0615425,8.7741344,1696
-3185196,53.0590256,8.7780123,1697
-3185196,53.0609753,8.7848701,1698
-3185196,53.056284,8.7839847,1699
-3185196,53.0541823,8.7854235,1700
-3185196,53.0538966,8.7915421,1701
-3185196,53.056006,8.7953486,1702
-3185196,53.0465167,8.8189275,1703
-3185196,53.0440043,8.8241022,1704
-3185196,53.0409061,8.8281429,1705
-3336378,53.051805,8.8189593,1706
-3336378,53.0821074,8.8135271,1707
-3336379,53.0516787,8.8193566,1708
-3336380,53.1734602,8.6464658,1709
-3373529,53.1710163,8.6324922,1710
-3373529,53.1853598,8.6169006,1711
-3373529,53.1852427,8.5942087,1712
-3373529,53.183712,8.5806046,1713
-3373529,53.185488,8.5706177,1714
-3373529,53.1889315,8.5573298,1715
-3373529,53.1935036,8.5447688,1716
-3373529,53.2048843,8.5158145,1717
-3373530,53.1708252,8.6322796,1718
-3373530,53.172662,8.6522585,1719
-3373530,53.1714328,8.6712029,1720
-3373530,53.1707061,8.6882194,1721
-3373530,53.1596948,8.7048867,1722
-3373530,53.1350051,8.7425975,1723
-3373530,53.103367,8.7865402,1724
-3373530,53.0604558,8.8889177,1725
-3373530,53.0393023,8.9459948,1726
-4261048,53.5350531,8.5997911,1727
-4425378,53.0833147,8.8148343,1728
-4425378,53.0877023,8.9356592,1729
-4458994,53.0835459,8.8140657,1730
-4593209,53.1769784,8.6847747,1731
-4593209,53.1800358,8.7320639,1732
-5346093,53.567878,8.5653882,1733
-5346093,53.5693902,8.5604888,1734
-5346093,53.5731997,8.5602197,1735
-5346093,53.5811046,8.5478947,1736
-5346093,53.5959072,8.5397568,1737
-5513686,53.0865927,8.9403178,1738
-5513686,53.0840909,8.8127024,1739
-5626115,53.5350288,8.5995133,1740
-5626115,53.5088309,8.6115355,1741
-5754318,53.0821401,8.8105896,1742
-6129984,53.194396,8.6149912,1743
-6183796,53.1078054,8.7410882,1744
-6183797,53.096256,8.7460892,1745
-6251915,53.0719515,8.8030042,1746
-6251916,53.0845593,8.8167823,1747
-6251916,53.0862898,8.8181929,1748
-6251919,53.0866887,8.8183064,1749
-6251919,53.0843903,8.8166559,1750
-6251919,53.0614534,8.7868959,1751
-6253014,53.0847515,8.8509305,1752
-6559182,53.1907488,8.6354251,1753
-6559182,53.1906215,8.6311785,1754
-6559182,53.188182,8.6293942,1755
-6559182,53.1841665,8.6264094,1756
-6559182,53.1781634,8.6247429,1757
-6559182,53.1744102,8.623985,1758
-6559182,53.1697553,8.6283579,1759
-6559183,53.1752794,8.6242875,1760
-6559183,53.178893,8.6249299,1761
-6559183,53.1833566,8.6260986,1762
-6559183,53.182946,8.6343984,1763
-6559183,53.1858632,8.6376531,1764
-6559183,53.1885375,8.6406411,1765
-6560472,53.1947326,8.6150382,1766
-6560472,53.1925607,8.6171911,1767
-6560472,53.190696,8.6184409,1768
-6560472,53.1861406,8.6191371,1769
-6560472,53.1700892,8.6279496,1770
-6648747,53.1214837,8.7526593,1771
-6785438,53.1684836,8.6615385,1772
+19936,53.0487235,8.7423629,0
```

Comment:
@mehdisadeghi was this a bug? I just realized that the shape_id changed but the shape_pt_sequence kept growing in the previous output :(

Reply:
@carocad yes it was. Previously I was creating one giant shape out of all existing stops!

Reply:
oh good catch !!